packages/libs/components/src/map/MapVEuMap.tsx

```diff
@@ -443,7 +443,7 @@ function CustomZoomControl(props: CustomZoomControlProps) {
         <div className="leaflet-control-zoom leaflet-bar leaflet-control">
           <a
             className={'leaflet-control-zoom-in' + disableMaxZoomButton}
-            href="#"
+            href="/#"
```

Comment:
Could you explain these changes please?

I have played around and for some reason any changes I make don't affect the zoom controls on screen. Very odd.

Reply:
@bobular As I mentioned before, it was simply to fix ESLint complaints, not a change to fix something.